lib/fortnox/api/base.rb

```diff
@@ -37,7 +37,7 @@ def initialize
           execute do |remote|
             response = remote.send( method, *args )
             self.headers['Access-Token'] = get_access_token
-            return response
```

Comment:
Ah yes ... Silly mistake, since the return is in a block it will return not only the block but the entire function the block is in :)